_artifacts/document-attributes-global.adoc

```diff
@@ -62,7 +62,7 @@ endif::[]
 
 // generic entities
 :org-name: Red{nbsp}Hat
-:rhodsdocshome: https://access.redhat.com/documentation/en-us/{url-productname-long}/{vernum}/
+:rhoaidocshome: https://access.redhat.com/documentation/en-us/{url-productname-long}/{vernum}/
 :rhoaidocshome: https://access.redhat.com/documentation/en-us/{url-productname-long}/{vernum}/
```

Comment:
Also, I did a project-wide search of our docs, and we still mention {rhodsdocshome} in some of our modules. We will need to change these to {rhoaidocshome}, as the old :rhodsdocshome: attribute has now been removed with this PR.